cmd/mtaBuild.go

```diff
@@ -187,6 +187,23 @@ func runMtaBuild(config mtaBuildOptions,
 	}
 
 	commonPipelineEnvironment.mtarFilePath = mtarName
+
+	err = installMavenArtifacts(e)
+
+	return err
+}
+
+func installMavenArtifacts(e execRunner) error {
+	pomXMLExists, err := piperutils.FileExists("pom.xml")
+	if err != nil {
+		return err
+	}
+	if pomXMLExists {
+		err = maven.InstallMavenArtifacts(e, maven.EvaluateOptions{})
```

Comment:
No, because of how the mta build works. If the options are set, it will copy the files into the right places, so maven will pick them up without configuration. See `handleSettingsFiles` and the methods it calls for further reference.

Reply:
OK, but doesn't that violate the principle of being able to run piper locally without overwriting stuff unexpectedly?

Reply:
Yes, but the mta builder provides no better option to us. Discussed with them, without a satisfying result. You could configure maven via the mta yaml file, but we don't want to hotpatch that either.

Reply:
Is that also true for the M2Path? Was only aware that ProjectSettingsFile and GlobalSettingsFile are copied to the m2 home. But also the local maven repository?

Reply:
Yes I missed the m2path. Added now. Thanks for the hint.